pkg/client/hostsubnets.go

```diff
@@ -47,7 +48,7 @@ func (c *hostSubnet) List(opts metainternal.ListOptions) (result *sdnapi.HostSub
 }
 
 // Get returns host subnet information for a given host or an error
-func (c *hostSubnet) Get(hostName string) (result *sdnapi.HostSubnet, err error) {
+func (c *hostSubnet) Get(hostName string, options metav1.GetOptions) (result *sdnapi.HostSubnet, err error) {
```

Comment:
Oh, VersionedParams, yes. My sed expected "name string" and hostName messed it up.